### PR TITLE
Fix PiCT reported bugs (4).

### PR DESCRIPTION
**Pull request by swoocn**
- Fix (PiCT) Seller Registration screen: combine Seller Details and Advanced Seller Settings

**[see commits]**

_This PR is subject to change if additional bugs are resolved before approval._

### CODE REVIEW

**Comment by swoocn** [comment]
Hi @tate1650 - Could you please keep this PR open and refrain from merging it until our PM has given his approval?  I will let you know when ready.  Thanks!

File: backend/tsconfig.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "compilerOptions": {
     "target": "es2016",
     "module": "commonjs",
-    "rootDir": "./src",
```
Comment (swoocn) [inline backend/tsconfig.json:5]: Unfortunately, your rootDir change from earlier broke the test build.  
Curious if the "./" change works for you with the Dockerfile..?

Comment (tate1650) [reply backend/tsconfig.json:5]: It does not. That was the main reason I changed it to `./src` in the first place.

However, after a little experimenting with some code changes, I realized my initial assessment of the `yarn build` command not creating a proper build with the `./` setting was wrong. It was just in a different location that `ecosystem.config.js` couldn't find. So I changed where the config file was looking for the `index.js` file and now things appear to be working. So I'll include that small change (which should only affect prod builds) in the next PR I put together to keep things up to date between our repos.

**Review by tate1650 — APPROVED** [review]
Everything here looks good. Gonna go ahead and approve/merge this.

**Pull request merged**